client/app/bundles/comments/store/routerCommentsStore.js

```diff
@@ -0,0 +1,29 @@
+import { applyMiddleware, compose, createStore, combineReducers } from 'redux';
+import thunkMiddleware from 'redux-thunk';
+import loggerMiddleware from 'libs/middlewares/loggerMiddleware';
+import reducers from '../reducers';
+import { routerReducer } from 'react-router-redux';
+import { initalStates } from '../reducers';
+
+export default props => {
+  const initialComments = props;
+  const { $$commentsState } = initalStates;
+  const initialState = {
+    $$commentsStore: $$commentsState.merge({
+      $$comments: initialComments,
+    }),
+  };
+
+  // https://github.com/reactjs/react-router-redux
+  const reducer = combineReducers({
+    ...reducers,
+    routing: routerReducer,
```

Comment:
Since this PR is already merged, feel free to remove on the next PR, @alexfedoseev